docs2/site/docs/migrations/migration6.md

```diff
@@ -13,6 +13,34 @@ memory usage is reduced by freeing unnecessary references after obtaining the re
 
 Particularly useful for authentication checks, now validation rules are asynchronous.
 
+### 3. Add `AddApolloTracing` builder method (added in 5.3.0)
+
+This method adds the `InstrumentFieldsMiddleware` to the schema, and conditionally enables metrics
+during execution via `ExecutionOptions.EnableMetrics`. It also appends the Apollo Tracing results
+to the execution result if metrics is enabled during execution.
+
+### 4. Add `ConfigureExecution` builder method (added in 5.3.0)
```

Comment:
Why here for v6 migration?

Reply:
It is not listed in any other migration notes, and is worth mentioning.